posts/mpox-preparedness/index.qmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "2024 mpox outbreak: common analytics tasks and available R tools"
+author:
+  - name: "James Azam"
+    orcid: "0000-0001-5782-7330"
+  - name: "Hugo Gruson"
+    orcid: "0000-0002-4094-147"
+  - name: "Adam Kucharski"
+    orcid: "0000-0001-8814-9421"
+date: "2024-01-02"
+categories: [R, Epiverse-TRACE, mpox, outbreak, outbreak-analytics, DOI]
+bibliography: index.bib
+format:
+  html: 
+    toc: true
+---
+
+There is an ongoing outbreak of mpox globally and countries are on the high alert. The Democratic Republic of Congo is so far the worst hit with a total of 7, 851 cases and 384 deaths between January 1 and May 26, 2024 [^1]. Before 2022, there were no reports of sustained mpox outbreaks in most countries of the world. However, within 1 year (Jan 1, 2022, and Jan 29, 2023), 110 countries in all six WHO Regions had reported a total of 85, 473 confirmed cases and 89 deaths [@laurenson2023description].
+
+Mpox is a highly contagious and severe disease that is transmitted through respiratory droplets and direct contact with infected persons. The disease is characterized by fever, cough, and a rash and has an incubation period between 3 - 34 days, depending on the transmission route [@gessain2022monkeypox]. Infected individuals may experience severe symptoms leading to hospitalisation or death. There are two genetic clades: clade I and clade II, which also has subclades IIa and IIb [@laurenson2023description].
```

Comment:
```suggestion
Mpox is transmitted through respiratory droplets and direct contact with infected persons. The disease is characterized by fever, cough, and a rash and has an incubation period between 3 - 34 days, depending on the transmission route [@gessain2022monkeypox]. Infected individuals may experience severe symptoms leading to hospitalisation or death. There are two genetic clades: clade I and clade II, which also has subclades IIa and IIb [@laurenson2023description].
```
I don't think there's evidence it's 'highly contagious' (R is below 1 in general population)? I'd either give the severity estimate (e.g. https://www.medrxiv.org/content/10.1101/2024.04.23.24306209v1) or cut.

For incubation period, I'd suggest using the Ward estimate (https://www.bmj.com/content/379/bmj-2022-073153) rather than the range quoted in a review.

Reply:
I'm now using the incubation period from the Ward paper in b91aa593cfc3dc034aa337083653682d252a9efd.